pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/con_about_pantheon.adoc

```diff
@@ -0,0 +1,36 @@
+[id='about-pantheon_{context}']
+= About Pantheon
+
+Pantheon is a content-management system and publishing platform developed by Red Hat.
+You can use it to publish documents based on AsciiDoc source.
+After adding your modules and assemblies to Pantheon, you can do the following:
+
+. Add metadata
+. Preview your content using the look and feel of the Customer Portal
+. Associate modules and assemblies with a product
+. Publish, and unpublish content
+
+The following table describes features that Pantheon does and does not provide.
+
+.Pantheon Features
+|===
+|Pantheon provides |Pantheon does not provide
+
+a|* A GUI for managing modules and assemblies, including the ability for authors to search for specific content and other workflow features that are more user-friendly than those currently available,
+* Metadata editing to optimize content for search engines
```

Comment:
No period here, feels inconsistent.

Reply:
Good catch. I added periods to all of the items. Only the very last cell (about source control) actually needs one, but I added to the rest for consistency.